### PR TITLE
S0.5: Add admin user calibration view

### PR DESCRIPTION
**Pull request by je-l**
NOTE: Don't merge yet.

Add interface for "admin mode" where additional button is shown on map - admin user can start a raspberry calibration where position for the devices is given.

The authentication and MQTT functionality is still missing, this has only UI implementation for the feature.

Files with mostly only moved lines:
* mqttConnection.ts
* mqttConnectionTest.ts

Important changes found in files:
* calibrationPanel.tsx
* router.tsx

### CODE REVIEW

File: client/src/visualization/mapContainer.tsx [hunk 1]
```diff
@@ -18,7 +17,12 @@ import Deserializer, {
...
 } from '../location/mqttDeserialize';
+import { useUbiMqtt } from '../location/mqttConnection';
 import BluetoothNameModal from './bluetoothNameModal';
+import raspberryLogo from '../../asset/rasp.png';
+import { RaspberryLocation } from './calibrationPanel';
+import { StaticUbiMarker, OfflineMarker, NonUserMarker } from './marker';
+import { Location } from '../common/typeUtil';
 
 const KUMPULA_COORDS = { lat: 60.2046657, lon: 24.9621132 };
 const DEFAULT_NONTRACKED_ZOOM = 12;
```
Comment (mriekkin) [inline client/src/visualization/mapContainer.tsx]: This has nothing in particular to do with this PR.

Wouldn't it be better if this default zoom value was larger, for instance 17.

File: client/src/visualization/calibrationPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  newName,
+  setNewName,
+  newHeight,
+  setNewHeight,
+}: Props) => (
+  <CalibrationBar>
+    <CalibrationHeader>Set Raspberry Pi locations</CalibrationHeader>
+    <InfoSection>
+      Click location on map and enter name and height in millimeters relative to
+      the second floor for the Raspberry Pi.
```
[the remark this answers is not in the document]
Comment (je-l) [reply client/src/visualization/calibrationPanel.tsx]: Your version is definitely better.. ill change this.

File: client/src/visualization/mapContainer.tsx (hunk 1 above)
Comment (je-l) [reply client/src/visualization/mapContainer.tsx]: Good idea

**Comment by je-l** [comment]
This branch has been merged to master but github closed this automatically for some reason.

**Pull request closed without merging**